### DISCUSSION TITLE
Add Glue42 support to desktopJS

### DISCUSSION THREAD

**Discussion by psmulovics** [morganstanley/desktopJS#472] [Ideas]

**Comment by djasnowski** [comment]
What kind of discussion?

**Comment by bingenito** [comment]
Hi @naknode .  The discussions were progressing a bit offline and then I would summarize here.  The current plan is the concrete implementation would be done outside this monorepo and within their own repository.  There would be a list/links in the readme here to these 3rd party implementations and other extensions that form the whole ecosystem.  It opens up a larger discussion around possibly breaking all but the default implementations from here into separate repositories too for consistency.  This is at the possible expense of ease of development a monorepo can provide so we should think through the possibilities.

**Comment by djasnowski** [comment]
@bingenito Thanks for explaining. Can't wait to see future development on this. I'd like to help out more now that I do TypeScript at my new job.
